### PR TITLE
getConfigEntry tweaks

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- make function recognize SQF expressions that can be read using getNumber
- support keyword "STRING" and "TEXT"
- header fixes
- standard variable names

### CODE REVIEW

**Comment by Dystopian** [comment]
Don't you want to remove `_return` variable and return from `switch` directly to slightly speedup the function?

File: addons/common/fnc_getConfigEntry.sqf [hunk 1]
```diff
@@ -28,24 +28,28 @@ SCRIPT(getConfigEntry);
...
 params ["_configEntry", "_entryType", "_defaultValue"];
 
-private _r = _defaultValue;
-
-switch (toLower _entryType) do {
-    case "text": {
-        if (isText _configEntry) then {
-            _r = getText _configEntry;
-        };
+if (toUpper _entryType in ["STRING", "TEXT"]) exitWith {
```
Comment (PabstMirror) [inline addons/common/fnc_getConfigEntry.sqf]:
```suggestion
if (_entryType == "STRING" || _entryType == "TEXT") exitWith {
```

Comment (bux) [reply addons/common/fnc_getConfigEntry.sqf]: no `toUpper`?

Comment (PabstMirror) [reply addons/common/fnc_getConfigEntry.sqf]: `==` isn't case sensitive

Comment (Dystopian) [reply addons/common/fnc_getConfigEntry.sqf]: @PabstMirror's code is a little faster

**Comment by Dystopian** [comment]
tested, works good

**Pull request merged**